From 2c89190b01d04228c668ffbe0a379953c0ba5fd9 Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Fri, 30 May 2014 09:14:43 +0800
Subject: [PATCH 55/95] ENGR00133115 CTS Media JetPlayerTest fail

After LoadFromFd test finished, same JetPlayer object is used in
following LoadFromPath test, but data in audioTrack from previous
test is not yet outputted completely, it affect second test and leads
to audioTrack->write() fail.
Add a workaround to let all data of first test be outputted.

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 media/libmedia/JetPlayer.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/media/libmedia/JetPlayer.cpp b/media/libmedia/JetPlayer.cpp
index 34deb59..efcadfa 100644
--- a/media/libmedia/JetPlayer.cpp
+++ b/media/libmedia/JetPlayer.cpp
@@ -362,7 +362,9 @@ int JetPlayer::loadFromFD(const int fd, const long long offset, const long long
 int JetPlayer::closeFile()
 {
     Mutex::Autolock lock(mMutex);
-    return JET_CloseFile(mEasData);
+    int ret = JET_CloseFile(mEasData);
+    usleep(100000);
+    return ret;
 }
 
 
-- 
1.8.0

